src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -292,4 +292,35 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Given there is a disabled product :name
+     */
+    public function thereIsDisabledProduct($name)
+    {
+        $this->thereIsProduct($name, false);
+    }
+
+    /**
+     * @Given there is an enabled product :name
+     */
+    public function thereIsEnabledProduct($name)
+    {
+        $this->thereIsProduct($name, true);
+    }
+
+    /**
+     * @Given /^There is a product named "([^""]*)"$/
```

Comment:
I'm not sure about this step (and related steps above)... "There is a product ..." suggests that we're creating product in this step (e.g. like "there are products" step). What we actually do, is retrieving it by repository, which can be confusing.
What do you thing about ``* @Given /^Product "([^""]*)" is (disabled|enabled) $/``?